src/components/Form/FormControl.tsx

```diff
@@ -32,7 +32,10 @@ type typeType =
   | 'week'
   | undefined;
 
-type FormControlElement = HTMLInputElement | HTMLTextAreaElement;
+type FormControlElement =
+  | HTMLInputElement
+  | HTMLTextAreaElement
+  | Pick<HTMLInputElement, 'min'>;
```

Comment:
do we still need this (line 38)?

Reply:
We don't. I've removed it in the latest commit